src/commands/eval.js

```diff
@@ -127,11 +134,11 @@ Output format can be specified.
 `
 
 EvalCommand.examples = [
-  '$ fauna eval dbname "Paginate(Classes())"',
-  '$ fauna eval dbname --file=/path/to/queries.fql',
-  '$ echo "Add(1,1)" | fauna eval dbname --stdin',
-  '$ fauna eval dbname "Add(2,3)" --output=/tmp/result"',
-  '$ fauna eval dbname "Add(2,3)" --format=json --output=/tmp/result"',
+  '$ fauna eval "Paginate(Classes())"',
```

Comment:
@marrony Would it be appropriate for us to also add this command and its examples to the repo's `README`?

Reply:
added